### PR TITLE
Reopen slaw/scot jets bounty

### PR DESCRIPTION
**Pull request by thelifeandtimes**
@mikeosborne is making completed as true the right approach for this? @sigilante noted that it is not going to happen and may be irrelevant as New Mars becomes a thing.

### CODE REVIEW

**Comment by joemfb** [comment]
This bounty is still relevant.

**Comment by thelifeandtimes** [comment]
Dope, then @mikeosborne lets just update the metadata to reopen this to applicants.

**Comment by mikeosborne** [comment]
> This bounty is still relevant.

@sigilante thought the issue would go away or change with the new design. @joemfb happy to reopen, let me know either way

**Comment by sigilante** [comment]
For context:  I rewrote the jets from scratch twice using two different approaches, but the memory leak transiently appeared no matter what I did with those.  After a lot of hours I decided it was a sunk cost & that whatever I had done wasn't striking at the root memory leak issue.

(I have the jet code & it wouldn't take long to get it in the binary.  So it speeds things up BUT it doesn't solve the main issue it was supposed to solve w.r.t. memory issue.)

The relevance comment was about what happens to jets with New Mars moving away from u3, when/if that happens.

**Comment by mikeosborne** [comment]
Still haven't heard back from @joemfb - I'll just go on his last comment and reopen the bounty.

File: content/grants/slaw-scot-jets.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ description = "Tune slaw/scot jets for better perfomance."
 reward = "1 star"
 mentor = "~timluc-miptev"
 assignee = "~lagrev-nocfep"
```
Comment (mikeosborne) [inline content/grants/slaw-scot-jets.md:14]:
```suggestion
assignee = ""
```

File: content/grants/slaw-scot-jets.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ description = "Tune slaw/scot jets for better perfomance."
 reward = "1 star"
 mentor = "~timluc-miptev"
```
Comment (mikeosborne) [inline content/grants/slaw-scot-jets.md:13]:
```suggestion
mentor = "~master-morzod"
```

File: content/grants/slaw-scot-jets.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ description = "Tune slaw/scot jets for better perfomance."
...
 assignee = "~lagrev-nocfep"
-completed = false
+completed = true
```
Comment (mikeosborne) [inline content/grants/slaw-scot-jets.md:15]:
```suggestion
completed = false
```

**Comment by mikeosborne** [comment]
I'm just going to close this one and create a new one to reopen the bounty

**Pull request closed without merging**